ts/DataGrid/Defaults.ts

```diff
@@ -70,7 +70,8 @@ namespace Defaults {
             },
             rows: {
                 bufferSize: 10,
-                strictHeights: false
+                strictHeights: false,
+                virtualization: true
```

Comment:
I don't have a specific idea, but I'm a bit puzzled by the fact that we have options that are only important when virtualization is enabled (e.g. bufferSize) and whether we shouldn't somehow combine them with that. But that's a loose idea.

I would also like to add the `tableLayout` option with the values ​​`auto` or `fixed`, which will always be `fixed` for a virtualized table, and `auto` by default for a non-virtualized one (concerns pt 3).